test-fixtures/src/main/kotlin/org/gradle/kotlin/dsl/fixtures/Testing.kt

```diff
@@ -19,14 +24,20 @@ fun <T : Throwable> assertFailsWith(exception: KClass<out T>, block: () -> Unit)
     throw IllegalStateException()
 }
 
+
 inline
 fun <reified T> withInstanceOf(o: Any, block: T.() -> Unit) {
     block(assertInstanceOf<T>(o))
 }
 
+
 inline
 fun <reified T> assertInstanceOf(o: Any): T {
     assertThat(o, instanceOf(T::class.java))
     return o as T
 }
 
+
+fun classLoaderFor(vararg classPath: File): URLClassLoader =
+    URLClassLoader.newInstance(
+        classPath.map { it.toURI().toURL() }.toTypedArray())
```

Comment:
`URLClassLoader` has some new behaviour under JDK 9. Has this been tested with JDK 9?

Reply:
The whole test-suite is continuously tested against JDK 9.

Reply:
👍